workflow/play.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash 
+
+set -eou pipefail
+
+if [ $# -ne 2 ] 
+then
+	echo "usage: $0 <workflow_name> <resource_name>"
+	exit 1
+fi
+
+./increment_index.sh $@
```

Comment:
I'd rebase on/merge `master` - it runs `shellcheck` on things now via CI (and has `make shellcheck` for local). 

(I'm 99% sure it's going to want this quoted)